ludwig/models/modules/sequence_decoders.py

```diff
@@ -149,104 +149,56 @@ class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
-            regularize=True,
+            use_bias=True,
+            kernel_initializer='glorot_uniform',
+            bias_initializer='zeros',
+            kernel_regularizer=None,
+            bias_regularizer=None,
+            activity_regularizer=None,
             attention=False,
             num_classes=0,
-            loss=None,
+            is_timeseries=None,
             **kwargs
     ):
         super(SequenceTaggerDecoder, self).__init__()
         self.initializer = initializer
-        self.regularize = regularize
         self.attention = attention
 
-        # todo tf2 add other required parameters for Dense layer
-        self.decoder_layer = Dense(num_classes)
+        if is_timeseries:
+            units = 1
+        else:
+            units = num_classes
+
+        self.decoder_layer = Dense(
+            units,
+            use_bias=use_bias,
+            kernel_initializer=kernel_initializer,
+            bias_initializer=bias_initializer,
+            kernel_regularizer=kernel_regularizer,
+            bias_regularizer=bias_regularizer,
+            activity_regularizer=activity_regularizer
+        )
 
     def call(
             self,
-            hidden,
-            **kwargs
-            # output_feature,
-            # targets,
-            # hidden_size,
-            # regularizer,
-            # is_timeseries=False
+            inputs,
+            training=None,
+            mask=None
     ):
-        logger.debug('  hidden shape: {0}'.format(hidden.shape))
-        if len(hidden.shape) != 3:
+        logger.debug('  hidden shape: {0}'.format(inputs.shape))
```

Comment:
I'm removing all this loggig calls from call() everywhere for now

Reply:
done